test/unit/cli_test.rb

```diff
@@ -18,34 +18,26 @@ class CliTest < Minitest::Test
 
     FakeResult = Struct.new(:ok?, :error_value)
 
-    class FakeOffense < Packwerk::Offense
-      def initialize; end
-    end
-
-    class FakeRunContext < Packwerk::RunContext
-      def initialize; end
-    end
-
-    class FakeConfiguration < Packwerk::Configuration
-      def initialize; end
-    end
-
```

Comment:
new style guide rules rightly rejected this; it doesn't initialize the objects correctly. The fix was a little more involved but I discovered that actually non of these fake classes are necessary.